src/geo_templates/default/main_page/extra_pages/advertise_with_us.tpl

```diff
@@ -1,39 +1,29 @@
-<div class="content_box">
-	<h1 class="title">Advertise With Us</h1>
-	    <p>Please <a href="index.php?a=28&b=136" title="Contact Us" style="text-decoration:underline; font-weight: bold;">Click Here</a> 
-			to request more details on how to advertise on one of the fastest growing sites on the web!</p> 
-
-		<div class="advertise">
-			<div>
-				<a href="http://lunarpages.com/id/geodesic/goto/basic" onmouseover="window.status='http://www.lunarpages.com';return true" onmouseout="window.status='';return true" onclick="window.open(this.href); return false;">
-					<img style="border: medium none ;" src="{external file='images/banners/lunarpages_468x60.gif'}" alt="Lunarpages.com Web Hosting" title="Lunarpages.com Web Hosting">
-				</a>
-			</div>
-			<div>
-				<a href="index.php?a=28&amp;b=144" title="Website Name">
-					<img src="{external file='images/banners/banner1_468x60.jpg'}" alt="Advertise with Us" title="Advertise with Us" width="468" height="60" />
-				</a>
-			</div>
-			<div>
-				<a href="index.php?a=28&amp;b=144" title="Website Name">
-					<img src="{external file='images/banners/banner2_468x60.jpg'}" alt="Advertise with Us" title="Advertise with Us" width="468" height="60" />
-				</a>
-			</div>
-			<div>
-				<a href="index.php?a=28&amp;b=144" title="Website Name">
-					<img src="{external file='images/banners/banner3_468x60.jpg'}" alt="Advertise with Us" title="Advertise with Us" width="468" height="60" />
-				</a>
-			</div>
-			<div>
-				<a href="index.php?a=28&amp;b=144" title="Website Name">
-					<img src="{external file='images/banners/banner4_468x60.jpg'}" alt="Advertise with Us" title="Advertise with Us" width="468" height="60" />
-				</a>
-			</div>
-			<div>
-				<a href="index.php?a=28&amp;b=144" title="Website Name">
-					<img src="{external file='images/banners/banner5_468x60.jpg'}" alt="Advertise with Us" title="Advertise with Us" width="468" height="60" />
-				</a>
-			</div>
-		</div>
-
-</div>
\ No newline at end of file
+<div class="content_box">
+	<h1 class="title">Advertise With Us</h1>
+	    <p>Please <a href="index.php?a=28&b=136" title="Contact Us" style="text-decoration:underline; font-weight: bold;">Click Here</a>
+			to request more details on how to advertise on one of the fastest growing sites on the web!</p>
+
+		<div class="advertise">
+			<div>
+        		<a href="https://geodesicsolutions.org">
+		        	<img style="border: medium none ;" src="{external file='images/banners/sample_468x60.jpg'}" alt="Sammple Ad Banner" title="Sammple Ad Banner">
+		        </a>
+			</div>
+			<div>
+        		<a href="https://geodesicsolutions.org">
+		        	<img style="border: medium none ;" src="{external file='images/banners/sample_468x60.jpg'}" alt="Sammple Ad Banner" title="Sammple Ad Banner">
```

Comment:
Did you mean to also copy in that border style from the first one?  That goes for all of the locations, the style got copied into all of them.

Reply:
I'll fix it now. Standby.